scripts/sct_get_centerline.py

```diff
@@ -12,39 +12,39 @@
 from spinalcordtoolbox.centerline import optic
 import spinalcordtoolbox.image as msct_image
 from spinalcordtoolbox.image import Image
-from msct_types import Centerline
-from sct_viewer import ClickViewerPropseg
-from sct_straighten_spinalcord import smooth_centerline
+from sct_process_segmentation import extract_centerline
 
 
-def viewer_centerline(image_input_reoriented, interslice_gap, verbose):
+def _call_viewer_centerline(fname_in, interslice_gap=10):
```

Comment:
i find it a bit too dense now, how about setting a compromise with `interslice_gap=20`?

Reply:
I agree!